### ISSUE TITLE
play videos, forcing the fullscreen on Safari of the iPhone

### ISSUE THREAD

**Issue by ichengde** [facebook/react-vr#196]
### Description

Bug:
    **play videos, forcing the fullscreen on Safari of the iPhone**
    ```
        <VideoPano source={this.state.videoSource} playerState={this.state.playerState} style={this.state.videoPanoStyle}/>
    ```
1. iOs 10.x.x version:
    It plays videos, forcing the fullscreen on Safari of the iPhone.
    
2. iOS 9.x.x version:
    It can not be used, showing a white screen.

I noticed that GitHub code be updated but React VR code version of npm package is previous.
How find a good way to start and debug the latest code in addition to using npm to push a package? 
I tried to modify package.json using [Local Paths](https://docs.npmjs.com/files/package.json#local-paths) in dependencies to build React VR application,
but fail in resolve modules.

Is it planned to be compatible with the most non VR equipment?

#### Expected behavior
It makes videos playable inline on the iPhone (prevents automatic fullscreen).
It uses videos as WebGL/ThreeJS textures.

#### Actual behavior
It plays videos, forcing the fullscreen on Safari of the iPhone.

### Reproduction
[a minimal example](https://gist.github.com/ichengde/318232b8928ce1f4d8dff6aa95596926)

### Additional Information
* React VR version:
    "react": "~15.4.1",
    "react-native": "~0.42.0",
    "react-vr": "^1.1.0",
    "react-vr-web": "^1.1.0",
* Server Operating System: Windows 7 (x64)
* Graphics Card: [FILL THIS OUT: NVIDA, ATI, Intel? Which Driver?]
* Browser: Safari
* VR Device: Out of VR [iPhone 7 iOS 10.3.1]

**Comment by ichengde** [comment]
maybe use [iphone-inline-video](https://www.npmjs.com/package/iphone-inline-video) in ReactVR/js/Video/VRVideoPlayer.js to resolve this.

**Comment by larrylin28** [comment]
#1c46c4e will fix this issue on iPhone iOS 10+.
If you want to also support iPhone iOS 10-, you may still need to use something like [iphone-inline-video](https://www.npmjs.com/package/iphone-inline-video). Check https://facebook.github.io/react-vr/docs/video-player.html for how to write a customized video player using 3rd party libraries and hook it into React VR.

**Comment by larrylin28** [comment]
> I noticed that GitHub code be updated but React VR code version of npm package is previous.
How find a good way to start and debug the latest code in addition to using npm to push a package?
I tried to modify package.json using Local Paths in dependencies to build React VR application,
but fail in resolve modules.
@andrewimm, what would be the best way to test latest code?

**Comment by larrylin28** [comment]
> I noticed that GitHub code be updated but React VR code version of npm package is previous.
How find a good way to start and debug the latest code in addition to using npm to push a package?
I tried to modify package.json using Local Paths in dependencies to build React VR application,
but fail in resolve modules.

8681fc1 provides the way to test main build.

**Comment by ichengde** [comment]
#### Tnx! I'm using it.
talk about iOS 9.x.x version bug. It can not be used, showing a white screen. It is not the reason for playing the video. It seems to be wrong but the plan may not support iOS 9?
![_20170518172811](https://cloud.githubusercontent.com/assets/4899969/26196562/ea1d2fe4-3bf1-11e7-92cf-33b188382448.jpg)

**Comment by andrewimm** [comment]
Fixed in 1.2.0 release.
